buildlib/build/log.d

```diff
@@ -0,0 +1,34 @@
+module build.log;
+
+// TODO: possibly makethe log output File configurable (not just stdout)
+
+private __gshared bool verboseEnabled = false;
+void enableVerbose()
+{
+    verboseEnabled = true;
+}
+
+/**
+Logs message to stdout `enableVerbose` was called.
+Params:
+    args = the data to write to the log
+Note: marked inline so the caller doesn't have to generate arguments if verbose is disabled
+*/
+pragma(inline)
+void verbosef(T...)(string fmt, T args)
+{
+    import std.stdio;
+    if (verboseEnabled)
+        writefln(args);
+}
+
+/**
+Logs the message to stdout.
+Params:
+    args = the data to write to the log
+*/
+void logf(T...)(T args)
```

Comment:
What's the point of this? And why not `alias ...`?

Reply:
Allows you to edit all log statements in one place.  I might add a prefix to all log statements for example, or direct output to a different file.